ls1intum/Artemis
# Enable HTML5 push state routes

### Checklist

### Motivation and Context
So far all Artemis route use `/#/foo` style URLs. Angular and JHipster switched to HTML5 push state as default some while ago (e.g. jhipster/generator-jhipster#9098).

### Description
Update Setup to use HTML5 push state instead.
Forward old URLs to preserve links.

### Steps for Testing
1. Log in to Artemis
2. Check that all functionality is working as is.
3. Test that deep-links continue to work.
